jerry-core/ecma/operations/ecma-regexp-object.h

```diff
@@ -30,6 +30,8 @@
 
 /**
  * RegExp flags
+ * Note:
+ *      This enum has to be kept in sync with jerry_regexp_flags_t.
```

Comment:
Please add a STATIC_ASSERT to the `jerry.c` like we did for `jerry_error_t` and `jerry_init_flag_t`.